p2p/peer_error.go

```diff
@@ -86,6 +88,7 @@ var discReasonToString = [...]string{
 	DiscSelf:                "connected to self",
 	DiscReadTimeout:         "read timeout",
 	DiscSubprotocolError:    "subprotocol error",
+	DiscInvalid:             "invalid disconnect reason",
```

Comment:
@fjl  I think this bumps the length of the `discReasonToString` to 256... So it means that the `String()` func will never return the `uknown reason` error. it will return empty string instaed

Reply:
Good point, thanks!